### ISSUE TITLE
TypeError: node_adapter_1.default is not a function

### ISSUE THREAD

**Issue by CatsMiaow** [GoogleChromeLabs/comlink#476]
- Node.js v12.16.3
- TypeScript v3.9.5

`main.ts`
```ts
import { Worker } from 'worker_threads';
import * as comlink from 'comlink';
import nodeEndpoint from 'comlink/dist/umd/node-adapter';

const worker = new Worker(`${__dirname}/sub.js`);
comlink.wrap(nodeEndpoint(worker));
```
`Error`
```sh
/home/test/dist/worker/comlink/main.js:7
comlink.wrap(node_adapter_1.default(worker));
                                   ^

TypeError: node_adapter_1.default is not a function
    at Object.<anonymous> (/home/test/dist/worker/comlink/main.js:7:36)
    at Module._compile (internal/modules/cjs/loader.js:1133:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1153:10)
    at Module.load (internal/modules/cjs/loader.js:977:32)
    at Function.Module._load (internal/modules/cjs/loader.js:877:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:74:12)
    at internal/main/run_main_module.js:18:47
```

`node_modules/comlink/dist/umd/node-adapter.d.ts`
```ts
export default function nodeEndpoint(nep: NodeEndpoint): Endpoint;
```
`node_modules/comlink/dist/umd/node-adapter.js`
```ts
return nodeEndpoint;
```

It is exported by default, but the actual dist file is not.
> [TypeScript Playgound](https://www.typescriptlang.org/play/?target=99&module=3&ssl=4&ssc=1&pln=1&pc=1#code/KYDwDg9gTgLgBAE2AMwIYFcA29noHYDGMAlhHnHhEgKJ4KTF4wAUAlHAN4BQcvcUwGOih4A3FwC+XIA)

I like I could export and use as follows rather than importing it separately.
```ts
import { Worker } from 'worker_threads';
import * as comlink from 'comlink';

const worker = new Worker(`${__dirname}/sub.js`);
comlink.wrap(comlink.nodeEndpoint(worker));
```

**Comment by surma** [comment]
You are importing an UMD module as an ES module. If you use `comlink/dist/esm/node-adapter.mjs` it should hopefully work.

**Comment by CatsMiaow** [comment]
I have already tried. TypeScript cannot load `mjs` extension.
If there is no extension, it will try to load the `js` file.
> https://github.com/microsoft/TypeScript/issues/27957

When compiling TypeScript, the default function is exported to the default object, but Rollup doesn't seem to be.
> See https://github.com/rollup/rollup/issues/1156

This issue was fixed by adding `esModuleInterop` to `true` in `tsconfig.json` compilerOptions.

https://github.com/GoogleChromeLabs/comlink/blob/master/src/node-adapter.ts#L31
In my opinion, it would be better not to export `nodeEndpoint` as `default`.
